app/src/main/res/layout/fragment_balance.xml

```diff
@@ -301,6 +265,83 @@
                 app:layout_constraintEnd_toEndOf="parent"
                 app:layout_constraintTop_toTopOf="@id/wallet_icon"
                 />
+
+            <include
+                android:id="@+id/verified_wallet_chip"
+                layout="@layout/verified_wallet_chip"
+                android:layout_width="wrap_content"
+                android:layout_height="match_parent"
+                android:layout_marginTop="8dp"
+                android:visibility="gone"
+                app:layout_constraintStart_toStartOf="@id/active_wallet_address"
+                app:layout_constraintTop_toBottomOf="@id/active_wallet_address"
+                tools:visibility="visible"
+                />
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:id="@+id/unverified_wallet_layout"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="8dp"
+                android:layout_marginTop="8dp"
+                android:layout_marginEnd="8dp"
+                android:background="@drawable/background_dark_grey"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/active_wallet_address"
+                >
```

Comment:
Since we have a custom layout for verified wallet I think it makes sense to have another for unverified ones.
This would remove the layout complexity by far